### DISCUSSION TITLE
What's putting the Mozilla Public License in my own source files?

### DISCUSSION THREAD

**Discussion by gutley** [mozilla-mobile/firefox-ios#12303] [Q&A] [answered]
I just noticed that the standard MPL is being added to any new source files I create in my own fork of Firefox. ie:

```
// This Source Code Form is subject to the terms of the Mozilla Public
// License, v. 2.0. If a copy of the MPL was not distributed with this
// file, You can obtain one at http://mozilla.org/MPL/2.0/
```

It never used to do this, so I guess something has changed. 

How do I stop it?

**Comment by lmarceau** [comment, marked as answer]
This has been there for a while under `Client.xcodeproj/xcshareddata/IDETemplateMacros.plist` [file](https://github.com/mozilla-mobile/firefox-ios/blob/main/Client.xcodeproj/xcshareddata/IDETemplateMacros.plist). If you want a different one you could modify it there or remove it in your branch.

**Reply by gutley** [reply]
Thanks - I think I did that a while back too, but maybe I overlooked a merge conflict in a merge with a load of conflicts. I wouldn't have noticed it as I've not needed to create any new files until yesterday.

One of these days I must get around to writing up all the pain points of forking the iOS Firefox. Not sure you guys would be interested in reducing them, but it might help others.
